src/plugins/data/server/search/strategies/es_search/es_search_strategy.ts

```diff
@@ -48,6 +48,7 @@ export const esSearchStrategyProvider = (
         };
         const body = await esClient.asCurrentUser.search(params, {
           signal: abortSignal,
+          ...transport,
```

Comment:
Looks like we updated the type for all search strategies but implemented only for this search strategy. 
I think we should either implement it for all strategies or type only so that it can be used for this strategy. 
WDYT?